src/main/resources/static/js/MainWindow.js

```diff
@@ -887,7 +887,7 @@ function importData() {
 						var obj = {}, obs = [];
 						currentInput = $(this).find("select option:selected")
 								.val();
-						if (currentInput === null || currentInput === "") {
+						if (currentInput == null || currentInput == "") {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)